dandischema/models.py

```diff
@@ -979,12 +1042,12 @@ class BareAsset(CommonModel):
 
     wasDerivedFrom: Optional[List[BioSample]] = Field(None, nskey="prov")
     wasAttributedTo: Optional[List[Participant]] = Field(
-        None, description="Participant(s) to which this file belongs to", nskey="prov"
+        None, description="Participant(s) to which this file belongs to.", nskey="prov"
```

Comment:
"belongs to" is odd here. The participant owns the file? maybe "participant associated with this file."